doc/user_guide/index.rst

```diff
@@ -82,38 +79,28 @@ PSyclone User Guide
     :maxdepth: 2
     :caption: Introduction
 
-    introduction
     getting_going
```

Comment:
In 'Getting Going', under 'run' we still assume that a PSyKAl DSL is being used:

![image](https://github.com/stfc/PSyclone/assets/17409212/c0add558-5b63-4720-ae80-39bc128fe3df)

Please could you update this text too?

Reply:
I just made a light update on this to make clear that we are talking about PSyKAl here. But I am planning to revisit this whole section in a separate PR.

Reply:
Great, thanks.